protobuf/options.proto

```diff
@@ -44,4 +44,7 @@ message Options {
     oneof schema_lock_acquire_timeout_opt {
         int32 schema_lock_acquire_timeout_millis = 6;
     }
+    oneof primary_replica_opt {
+        bool primary_replica = 7;
```

Comment:
I wonder if `primary_replica` is misleading because it could be switched off and the primary replica might still be used. What do you think of the following alternatives:

- `require_primary_replica`
- `allow_secondary_replica`

Or do you think we should stick with `primary_replica` for conciseness?

Reply:
Fixed in https://github.com/graknlabs/protocol/pull/110/commits/6d5a5e1b58d91fd001e06b2820363e7194f7fd3f.